rpc/src/v1/impls/parity.rs

```diff
@@ -481,4 +487,21 @@ impl<C, M, U, S> Parity for ParityClient<C, M, U> where
 	fn submit_work_detail(&self, nonce: H64, pow_hash: H256, mix_hash: H256) -> Result<H256> {
 		helpers::submit_work_detail(&self.client, &self.miner, nonce, pow_hash, mix_hash)
 	}
+
+	fn status(&self) -> Result<()> {
+		let has_peers = self.client.spec_name() == "developmentchain" || self.sync.status().num_peers > 0;
```

Comment:
Can't we have a more generic method to check if it's the `dev` chain? Or couldn't it be the same check as for the light implementation?

Reply:
Good point, was trying to look for any detection we do already, but haven't found anything.
In light we don't really have `spec_name` exposed, but I agree that current way is suboptimal.
I'll try to figure something out.

Reply:
@ngotchac addressed, could you have a look?